models/config.go

```diff
@@ -2,89 +2,21 @@ package models
 
 import (
 	"fmt"
-	"io/ioutil"
-	"os"
 )
 
 const (
-	testingDB string = "postgres://postgres:postgres@localhost:5432/go_stop_go_test?sslmode=disable"
-	devDB     string = "postgres://postgres:postgres@localhost:5432/go_stop_go?sslmode=disable"
-
-	testEnv string = "test"
-	devEnv  string = "dev"
-	prodEnv string = "prod"
-
-	devBcryptRounds  int = 1
-	prodBcryptRounds int = 12
+	dbUrlFmt string = "postgres://%s:%s@%s:%d/%s?sslmode=disable"
```

Comment:
I didn't know you could assign a formatted string like this, neat!

Reply:
ya its just a string, it just has the formatting codes in there, you can treat it like any other string really